src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -198,6 +200,10 @@ internal class CommandLineParameterParser
     {
         private const int MaxPipePathLengthLinux = 108;
         private const int MaxPipePathLengthMacOS = 104;
+        internal static int MaxNameLength = Platform.IsWindows ? ushort.MaxValue : (Platform.IsLinux ? MaxPipePathLengthLinux : MaxPipePathLengthMacOS) - Path.GetTempPath().Length;
```

Comment:
This is really long. Can you please put make `MaxNameLength` a getter instead with a backed field? So the calculation logic can be put in the getter, and the result can be cached by the field.

Reply:
Maybe just reformat?
```suggestion
        internal readonly static int MaxNameLength = 
            Platform.IsWindows
                ? ushort.MaxValue
                : (Platform.IsLinux ? MaxPipePathLengthLinux : MaxPipePathLengthMacOS) - Path.GetTempPath().Length;

```

Reply:
Convert MaxNameLength to method.
Fixed.